test/test_nn_quantized.py

```diff
@@ -34,6 +36,67 @@ def test_relu_api(self):
 
 
 class ModuleAPITest(TestCase):
+    @given(
+        batch_size=st.integers(1, 5),
+        in_features=st.integers(16, 32),
+        out_features=st.integers(4, 8),
+        use_bias=st.booleans(),
+    )
+    def test_dynamic_linear_api(self, batch_size, in_features, out_features, use_bias):
```

Comment:
If there will be a lot of dynamic quantization stuff, maybe we should create a separate test class. Maybe `DynamicQuantizationTest`?

Reply:
Will fix this.

Reply:
Updated.